tests/_support/Helper/Acceptance/WPForms.php

```diff
@@ -372,9 +372,6 @@ public function configureConvertKitSettingsOnForm($I, $wpFormID, $formName, $nam
 		$I->waitForElementVisible('div[data-connection_id="' . $connectionID . '"] .wpforms-provider-fields');
 
 		if ($formName) {
-			// Confirm that Forms are in ascending alphabetical order.
-			$I->checkSelectFormOptionOrder($I, '[name="providers[convertkit][' . $connectionID . '][list_id]"]');
```

Comment:
Introducing `optgroup` to the dropdown selector makes this test fail, as it can't seem to grab the right `option` value to perform the assertion with. It's the only place used so don't think it's vital to keep checking this.